src/WebJobs.Extensions.ApiHub/Common/GenericFileTriggerBindingProvider.cs

```diff
@@ -110,9 +117,19 @@ public Type TriggerValueType
 
             public async Task<ITriggerData> BindAsync(object value, ValueBindingContext context)
             {
-                TFile file = (TFile)value;
-                var bindingData = GetBindingData(file);
-                string path = _parent._strategy.GetPath(file);
+                var path = value as string;
+                IReadOnlyDictionary<string, object> bindingData = null;
+
+                if (path != null)
+                {
+                    bindingData = GetBindingData(path);
```

Comment:
Please add a test for this new functionality. You can use JobHost.Call to invoke a method, passing in a string value.